site/en/docs/extensions/reference/webRequest/index.md

```diff
@@ -180,7 +180,7 @@ permissions][9]. Moreover, only the following schemes are accessible: `http://`,
 `chrome-extension://`. In addition, even certain requests with URLs using one of the above schemes
 are hidden. These include `chrome-extension://other_extension_id` where `other_extension_id` is not
 the ID of the extension to handle the request, `https://www.google.com/chrome`, and other sensitive
-requests core to browser functionality. Also synchronous XMLHttpRequests from your extension are
+requests core to browser functionality. Also synchronous `fetch()` requests from your extension are
```

Comment:
As Joe said, a simple find/replace is in most cases not possible, e.g. in this case, fetch requests are never synchronous and XHR should be kept here.